.github/workflows/ci.yml

```diff
@@ -15,44 +15,85 @@ jobs:
   org-check:
     name: Check GitHub Organization
     if: github.repository_owner == 'a-scie'
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       - name: Noop
         run: "true"
   ci:
-    name: (${{ matrix.os }}) CI
+    name: ${{ matrix.name }} CI
     needs: org-check
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # N.B.: macos-13 is the oldest non-deprecated Intel Mac runner and macos-14 is the oldest
         # non-deprecated ARM Mac runner.
-        os: [ ubuntu-22.04, linux-arm64, macos-13, macos-14, windows-2022, windows-arm64 ]
+        include:
+          - os: ubuntu-24.04
+            name: Linux x86-64
+            cross-target: x86_64-unknown-linux-musl
+            docker-platform: linux/amd64
+          - os: ubuntu-24.04
+            name: Linux aarch64
+            cross-target: aarch64-unknown-linux-musl
```

Comment:
Over in ptex I included a non-cross-target ubuntu-24.04 shard to cover the x86_64-unknown-linux-gnu case. Although there is no production release for this target (only the musl static binary), it is almost certainly the daily driver for any Linux user hacking on the project (this includes me). As a result I want CI to make sure everything works under plain old glibc Linux.

Reply:
I added a glibc shard.